Model/Customer/SourceProvider/IdleFilterModifier.php

```diff
@@ -38,6 +38,7 @@ public function apply(AbstractDb $collection, WebsiteInterface $website): void
             'e.entity_id=cv.customer_id',
             null
         );
+        $collection->getSelect()->where('website_id = ?', $website->getId());
```

Comment:
We can use the addFieldToFilter method available on collection here:
`$collection->addFieldToFilter('website_id', $website->getId());`

We use 'where' at line .46  because it's a complex condition, and we use 'joinLeft' because the default 'join' method available on collection force inner joins.